cli/bpmetadata/cmd.go

```diff
@@ -47,27 +51,53 @@ func generate(cmd *cobra.Command, args []string) error {
 		return fmt.Errorf("error getting working dir: %w", err)
 	}
 
+	//read existing metadata.yaml, if present
+	bpObj := &BlueprintMetadata{}
+	f, _ := os.ReadFile(metadataFileName)
+	// if file is present, try to unmarshal it in BlueprintMetadata
+	if f != nil {
+		bpObj, _ = unmarshalMetadata(f)
```

Comment:
If a yaml is found, it should always be valid. Lets throw an error and exit.

Reply:
Additionally we should also validate GVK so we only process correct metadata versions.

Reply:
GVK check is valid. However, I don't think we can assume files are always going to be valid. It is possible that someone may make a syntactical an error while updating labels, variable groups etc.

Reply:
Added a GVK check